RELEASE.md

```diff
@@ -0,0 +1,17 @@
+Release type: minor
+
+This release add the ability to disable query validation by setting
+`validate_queries` to `False`
+
+```python
+import strawberry
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def hello(self) -> str:
+        return "Hello"
+
+
+schema = strawberry.Schema(Query, validate_queries=validate_queries)
+```
```

Comment:
I wonder if we should move this flag to schema.execute instead of enabling it globally.

It might be better for enabling/disabling validation for specific clients/queries since we usually instantiate the schema once 🤔

Reply:
I've moved this option to the .execute function :)